coderefs/coderefs.go

```diff
@@ -15,14 +15,19 @@ import (
 )
 
 func Run(opts options.Options) {
+	if len(opts.ProjKey) > 0 {
+		opts.Projects = append(opts.Projects, options.Project{
+			ProjectKey: opts.ProjKey,
+		})
+	}
 	dir := opts.Dir
 	absPath, err := validation.NormalizeAndValidatePath(dir)
 	if err != nil {
 		log.Error.Fatalf("could not validate directory option: %s", err)
 	}
 
 	log.Info.Printf("absolute directory path: %s", absPath)
-	ldApi := ld.InitApiClient(ld.ApiOptions{ApiKey: opts.AccessToken, BaseUri: opts.BaseUri, ProjKey: opts.ProjKey, UserAgent: "LDFindCodeRefs/" + version.Version})
+	ldApi := ld.InitApiClient(ld.ApiOptions{ApiKey: opts.AccessToken, BaseUri: opts.BaseUri, UserAgent: "LDFindCodeRefs/" + version.Version})
```

Comment:
Only used to get the project for flags. Changed the calling function to pass in the relevant project.